src/AlgoliaQueueWorker.cs

```diff
@@ -1,24 +1,24 @@
-﻿using CMS.Base;
-using CMS.Core;
+﻿using System;
+using System.Collections.Generic;
 
-using Kentico.Xperience.AlgoliaSearch.Models;
-using Kentico.Xperience.AlgoliaSearch.Services;
+using CMS.Base;
+using CMS.Core;
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
+using Kentico.Xperience.Algolia.Models;
+using Kentico.Xperience.Algolia.Services;
 
-namespace Kentico.Xperience.AlgoliaSearch
```

Comment:
Will this namespace change mean a major version bump for this library?

Reply:
Ah, I see the update in the csproj. I usually keep that version information in the `Directory.Build.props` 😀.